.nvmrc

```diff
@@ -1 +1 @@
-v20
\ No newline at end of file
+v20.10
```

Comment:
Yes. It is because next-test-api-route-handler needs that particular version:
https://github.com/danskernesdigitalebibliotek/dpl-go/pull/10/commits/45ff724c614edb8248decca36a16277f0fa2e9d6